iXBRLViewerPlugin/viewer/src/js/concept.js

```diff
@@ -14,52 +14,52 @@
 
 import $ from 'jquery'
 
-export function Concept(report, name) {
-    this._c = report.data.concepts[name];
-}
-
-/*
- * Return a space separated list of reference values, or the empty string if
- * the concept has none.
- */
-Concept.prototype.referenceValuesAsString = function() {
-    if (!this._c || !this._c.r) {
-        return "";
+export class Concept {
+    constructor(report, name) {
+        this._c = report.data.concepts[name];
     }
-    else {
-        return $.map(this._c.r, function (r,j) {
-            return $.map(r, function (p,i) { 
-                return p[1] 
-            }).join(" ")
-        }).join(" ");
-    }
-}
 
-Concept.prototype.references = function () {
-    if (!this._c || !this._c.r) {
-        return  [];
+    /*
+     * Return a space separated list of reference values, or the empty string if
+     * the concept has none.
+     */
+    referenceValuesAsString() {
+        if (!this._c || !this._c.r) {
+            return "";
+        }
+        else {
+            return this._c.r.map(
+                r => r.map(p => p[1]).join(" ")
```

Comment:
This is fine, but I think `return this._c.r.flatMap(r -> r.map(p => p[1])).join(" ")` would also work.  One fewer `join`, and intermediate arrays maybe marginally better than intermediate strings.

Reply:
Done in 7dbd3fbe8f86d74c2f119b0b2e8a78aa4b4420c3